### PR TITLE
Updated identity argument value name and added doc string(Fixes #6247)

### LINKED ISSUE #6247: `identity` argument value name on `ockam secure-channel-listener create` should be `IDENTITY_NAME`. The doc string should also be updated
## Current behavior

The `identity` argument value name on `ockam secure-channel-listener create` is `IDENTITY`.

## Desired behavior

The `identity` argument value name on `ockam secure-channel-listener create` should be `IDENTITY_NAME` and a doc string should be added with the contents `Name of the Identity that the secure-channel listener will use`.

## Implementation details

This is the argument that must be changed:

https://github.com/build-trust/ockam/blob/9c49e470a574d91854aa24628584eda4554aaad0/implementations/rust/ockam/ockam_command/src/secure_channel/listener/create.rs#L40-L41

---

<i>

Some tips for newcomers:
1. Create a new branch for the issue you are working on (don't push directly to your fork's `develop` branch)
1. Run `cargo fmt`. You can configure the editor to run it automatically everytime a file is saved
1. Run `cargo clippy` to make sure the code compiles + follows rust standards/best practices
1. You don't need to keep your branch updated with `develop` until there is a merge conflict. If you still want to update it, use `git rebase develop` instead of `gir merge develop` 

We love helping new contributors! ❤️
If you have questions or need help as you explore, please join us on [Discord](https://discord.gg/bewvnm6zqS). If you're looking for other issues to contribute to, please checkout our [good first issues](https://github.com/build-trust/ockam/labels/good%20first%20issue).

</i>

### PR DESCRIPTION
**Pull request by AnshKumar200**
## Current behavior

The `identity` argument value name on `ockam secure-channel-listener` create is `IDENTITY`.

## Proposed changes

identity argument value name on `ockam secure-channel-listener create` is updated from `IDENTITY` to  `IDENTITY_NAME`. The doc string is also added

Merging will fix :- https://github.com/build-trust/ockam/issues/6247
## Checks

- [x] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [x] All commits in this Pull Request follow the Ockam [commit message convention](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages).
- [x] There are no Merge commits in this Pull Request. Ockam repo maintains a linear commit history. We merge Pull Requests by rebasing them onto the develop branch. Rebasing to the latest develop branch and force pushing to your Pull Request branch is okay.
- [x] I have read and accept the Ockam Community [Code of Conduct](https://github.com/build-trust/.github/blob/main/CODE_OF_CONDUCT.md).
- [x] I have read and accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/.github/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam/blob/develop/.github/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam](https://github.com/build-trust/ockam) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam/edit/develop/.github/CONTRIBUTORS.csv) file in the github web UI and create a separate Pull Request, this will mark the commit as verified.

### CODE REVIEW

File: implementations/rust/ockam/ockam_command/src/secure_channel/listener/create.rs [hunk 1]
```diff
@@ -37,7 +37,8 @@ pub struct CreateCommand {
     #[arg(value_name = "VAULT", long, requires = "identity")]
     vault: Option<String>,
 
-    #[arg(value_name = "IDENTITY", long)]
+    //Name of the Identity that the secure-channel listener will use
```
Comment (adrianbenavides) [inline implementations/rust/ockam/ockam_command/src/secure_channel/listener/create.rs]: This should be `/// Name ...` (note the extra `/` and the space)

**Review by adrianbenavides — COMMENTED** [review]
Hey @AnshKumar200, thanks for submitting this PR. Looks good, but we need a small change to be applied before we can merge it 👍

File: implementations/rust/ockam/ockam_command/src/secure_channel/listener/create.rs (hunk 1 above)
Comment (AnshKumar200) [reply implementations/rust/ockam/ockam_command/src/secure_channel/listener/create.rs]: Ohh ,forgot to put the third `/`. Fixed it.

**Review by adrianbenavides — APPROVED** [review]
LGTM 🚂  Thanks @AnshKumar200!

**Comment by AnshKumar200** [comment]
Thanks for the help and review!🙂

**Review by metaclips — APPROVED** [review]

**Pull request merged**